components/calendar/src/gregorian.rs

```diff
@@ -110,10 +110,11 @@ impl DateTime<Gregorian> {
         hour: u8,
         minute: u8,
         second: u8,
+        fraction: u32,
```

Comment:
thought: yeah i'm not sure if we really need this, I think it's _fine_ but i suspect most won't need it? They can always mutate later

But no strong opinion either way